app/src/androidTest/java/ch/sdp/vibester/activity/CreateProfileActivityTest.kt

```diff
@@ -39,21 +41,36 @@ class CreateProfileActivityTest {
     }
 
     @BindValue @JvmField
-    val mockUsersRepo = mockk<DataGetter>()
+    val mockAuthenticator = mockk<FireBaseAuthenticator>()
 
-    private fun createMockInvocation(email: String) {
-        every {mockUsersRepo.createUser(any(), any(), any(), any())} answers {
-            lastArg<(String) -> Unit>().invoke(email)
-        }
+    private fun createMockAuthenticator() {
+        val mockUser = createMockUser()
+        every { mockAuthenticator.getCurrUser() } returns mockUser
+        every { mockAuthenticator.getCurrUID() } returns mockUser.uid
+    }
 
-        every { mockUsersRepo.getUserData(any(), any()) } answers {
-            secondArg<(User) -> Unit>().invoke(User())
-        }
+    private fun createMockUser(): FirebaseUser {
+        val email = "mockuser@gmail.com"
+        val uid = "mockuseruid"
```

Comment:
```suggestion
        val email = "u@u.c"
        val uid = "passwo"
```
Will takes less CI times if shorter